web3/app.html

```diff
@@ -0,0 +1,265 @@
+<script src="https://cdn.jsdelivr.net/npm/web3@latest/dist/web3.min.js"></script>
+
+<script>
+console.log("Script started")
+
+window.addEventListener('load', function () {
+  console.log("Loading window")
```

Comment:
Please remove all console.logs

Reply:
Was going to do it after we are sure no more changes are needed - are we sure we won’t be going with Biconomy for meta tx? That requires changes on the frontend and might just left this in that case